pyproject.toml

```diff
@@ -66,6 +65,7 @@ pytest-mock = "^3.10.0"
 types-PyYAML = "*" # only for development purposes no need to make installation req
 types-requests = ">=2.28.11.16 <3.0.0.0" # only for development purposes no need to make installation req
 apache-airflow = "^2.7.3"
+snowflake = "^0.5.1"
```

Comment:
hey the import error says to install snowflake-python-connector not snowflake. They are two different libraries. This shouldnt be a full req for brickflow but just for the operator.

Reply:
It should only be required as a library to run the workflow in bricks, not needed to be install locally